README.md

```diff
@@ -8,61 +8,79 @@ Head over to the [marketplace](https://www.contentful.com/marketplace/) and foll
-  - `deploy:staging` This script is not necessary at this point in time. We are currently working to add an official Contentful staging organization to house staging marketplace partner spaces and the respective staging app definition, for which this script can eventually point to.
-  - Additionally, ensure that the package version number is below 1.0.0. The version of 1.0.0 is reserved for initial release. 
-
-> [!NOTE]
-> **Please note that dependabot is enabled in this repo and will run automatically.**
-> While it will attempt to only upgrade dependencies without breaking changes, it is possible it will still introduce issues. Creating and maintaining comprehensive tests is critical to preventing code and build issues. See [Keeping dependencies up to date](#keeping-dependencies-up-to-date) for more details.
+    - Creates a build artifact(s). `"vite build"` is common for most apps without specific needs, though whatever tool or process you prefer is fine so long as it creates artifacts that can be referenced in the deploy step.
+  - `test`
+    - Once again `"vitest"` is common for many apps and should be considered if you do not have complicated tests. Having at least basic testing and coverage will ensure your app does not deploy if breaking changes are introduced by your work or dependabot (see below).
+  - `lint`
+    - Runs a script to lint your app's code. This is a required script to ensure code quality and consistency, and it is recommended to use [ESLlint](https://eslint.org/) in your app. See the Cloudinary app for an example [ESLint configuration](apps/cloudinary2/.eslintrc.json).
+  - Additionally, ensure that the package version number is below 1.0.0. The version of 1.0.0 is reserved for initial release.
+
+- Have a `LICENSE` file at the root of your app directory.
+
+- Use [TypeScript](https://www.typescriptlang.org/). We require this to ensure code quality and maintainability for the marketplace apps.
+
+- Ensure dependencies are up to date. The validation checks will provide warnings if a new app has outdated dependencies.
+  > [!NOTE] > **Please note that dependabot is enabled in this repo and will run automatically.**
+  > While it will attempt to only upgrade dependencies without breaking changes, it is possible it will still introduce issues. Creating and maintaining comprehensive tests is critical to preventing code and build issues. See [Keeping dependencies up to date](#keeping-dependencies-up-to-date) for more details.
+
+### After a new app is introduced
+
+After a new app has been introduced into the repo, the Contentful ecosystem team will make a couple of updates:
+
+- Add the `deploy` script
```

Comment:
I think it should stay here because the list above lists what the developer should include in their script when they open the PR. We (Contentful) add the deploy script later after the PR is merged once we get the marketplace app id